src/components/views/rooms/RoomHeader.js

```diff
@@ -47,13 +48,15 @@ module.exports = React.createClass({
         onSaveClick: React.PropTypes.func,
         onSearchClick: React.PropTypes.func,
         onLeaveClick: React.PropTypes.func,
+        onCancelClick: React.PropTypes.func,
```

Comment:
My mistake, this looks to be necessary. I think I was confused by the fact that it wasn't in prop types previously.